phpstan.neon.dist

```diff
@@ -50,6 +50,9 @@ parameters:
 		-
 			message: '#Call to an undefined method Doctrine\\Persistence\\ObjectManager::getConnection\(\)#'
 			path: src/Bridge/Doctrine/Common/Util/IdentifierManagerTrait.php
+		-
+			message: '#Call to an undefined method Symfony\\Component\\Validator\\Constraint::getHtmlPattern\(\)\.#'
```

Comment:
imo this is not a false positive, Contraint has not such a method only `Symfony\Component\Validator\Constraints\Regex` has it.

Reply:
Reverted, now checking with instanceof Regex